Doc/library/asyncio-eventloop.rst

```diff
@@ -58,10 +60,13 @@ an event loop:
    event loop.
 
    .. deprecated:: 3.10
-      Emits a deprecation warning if there is no running event loop.
-      In future Python releases, this function may become an alias of
-      :func:`get_running_loop` and will accordingly raise a
-      :exc:`RuntimeError` if there is no running event loop.
+      Deprecation warning is emitted if there is no current event loop.
+
+   .. note::
+      In Python versions 3.10.0--3.10.8 and 3.11.0 this function
+      (and other functions which used it implicitly) emitted a
+      :exc:`DeprecationWarning` if there was no running event loop, even if
+      the current loop was set.
```

Comment:
Should we mention that in 3.12 it will raise in that case? (You do mention that in the policy docs.)